### PR TITLE
Update add-a-printer-ubuntu.md

### PR DESCRIPTION
**Pull request by Seibz**
added brother printer instructions to ubuntu article

### CODE REVIEW

File: content/add-a-printer-ubuntu.md [hunk 1]
```diff
@@ -51,6 +51,42 @@ sudo usermod -aG lpadmin $USER
 
 And when adding the printer, select type of connection as **DNS-SD**, and then **Forward**, and then **ESC/P-R**.
 
+## Brother Printers
+
+Brother provides a driver installation tool for Linux users. Install the appropriate driver for your Brother printer by downloading this tool and running the installer with your printer model appended to the command.
+
+1. Click [this link](https://support.brother.com/g/b/productsearch.aspx?c=us&lang=en&content=dl) to download Brother's Driver Install Tool for Linux while searching for your appropriate printer.
```
Comment (ahoneybun) [inline content/add-a-printer-ubuntu.md:58]: I believe this should be "for the appropriate printer"

File: content/add-a-printer-ubuntu.md [hunk 2]
```diff
@@ -51,6 +51,42 @@ sudo usermod -aG lpadmin $USER
...
+
+2. Click the "Click here to download the tool" link on the Brother download page.
+
+3. Select Linux (deb) for your "OS Version", then click OK.
+
+4. Click "Agree to the EULA and Download".
+
+5. Choose the Save File option when prompted.
+
+6. Press <kbd>Super</kbd> + <kbd>T</kbd> to launch the Terminal application.
```
Comment (ahoneybun) [inline content/add-a-printer-ubuntu.md:68]: This is the shortcut for Pop not Ubuntu so we need to change that.

**Review by ahoneybun — CHANGES_REQUESTED** [review]

File: content/add-a-printer-ubuntu.md (hunk 1 above)
Comment (storyaddict) [reply content/add-a-printer-ubuntu.md:58]: I think either works, but I think "the appropriate" sounds better.

Comment (Seibz) [reply content/add-a-printer-ubuntu.md:58]: I would agree

File: content/add-a-printer-ubuntu.md (hunk 2 above)
Comment (Seibz) [reply content/add-a-printer-ubuntu.md:68]: We would, do we have a different super key image?

**Review by Seibz — COMMENTED** [review]
Looks good

File: content/add-a-printer-ubuntu.md (hunk 2 above)
Comment (storyaddict) [reply content/add-a-printer-ubuntu.md:68]: I've just been using Super or SUPER like we have here. That way it's agnostic.

**Review by storyaddict — APPROVED** [review]
Fixed wording issue.

**Review by ahoneybun — APPROVED** [review]

**Pull request merged**